core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -135,6 +138,9 @@ private[spark] class ExecutorAllocationManager(
   private val executor =
     ThreadUtils.newDaemonSingleThreadScheduledExecutor("spark-dynamic-executor-allocation")
 
+  // Metric source for ExecutorAllocationManager to expose internal status to MetricsSystem.
+  private[spark] val executorAllocationManagerSource = new ExecutorAllocationManagerSource
```

Comment:
The class is already `private[spark]`, so we don't need to declare this again here.